explainer.md

```diff
@@ -0,0 +1,802 @@
+
+Unbound objects are objects that do not represent a date or time tied to the actual concrete timeline. Imagine a note saying "be home by 14:45". That information is not tied to a specific point in time. You don't know where the note came from. It could have been written at any time in the past. No date attaches to it. Equally you can't tell what timezone it belongs in. It may not mean local time.
+
+Even information like "I was born on the 18th of November 1976 at 15:23:30" does not attach to a specific point in time. We still lack the location of where the speaker was born.
+
+In addition there are more abstract dates that are used frequently in civil discourse. Such as "Christmas is on the 25th of December". Which year? Or "I moved to Britain in August 2014". Which day in August?
+
+These objects all have in common is that they are not tied to a specific point on the timeline.
+
+### CivilDate
+
+`CivilDate` (and its siblings) represents a date and time corresponding to the requirements of ISO-8601.
+
+This means specifically that it uses the *proleptic Gregorian calendar* whose days are 24 hours
+long and begin and end at midnight.
+
+#### CivilDate() - constructor
+
+The constructor may only be called as such. It takes 3 numeric arguments.
+
+- `year` the Gregorian year
+- `month` the Gregorian month
+- `day` the Gregorian day of the month
+
+#### CivilDate.prototype.year : number
+
+The `.year` property represents the year of the `CivilDate`.
+
+#### CivilDate.prototype.month : number
+
+The `.month` property represents the month of the `CivilDate`.
+
+#### CivilDate.prototype.day : number
+
+The `.day` property represents the day of the month of the `CivilDate`.
+
+#### CivilDate.prototype.dayOfWeek : number
+
+The `.dayOfWeek` property represents the day of the week where Monday is `1` and `Sunday` is `7` in accordance with ISO-8601.
+
+#### CivilDate.prototype.dayOfYear : number
+
+The `.dayOfYear` property represents the ordinal day of the Gregorian year according to ISO-8601.
+
+#### CivilDate.prototype.weekOfYear : number
+
+The `.weekOfYear` property represents the ISO week-number. Beware that dates at the begining of a year may be part of a week from the preceding year, and dates at the end of a year may be part of a week at the beginning of the next year, as the first week of any year is defined as the week that contains the first Thursday of the week.
+
+#### CivilDate.prototype.plus(value: IntervalLike) : CivilDate
+
+Creates a new `CivilDate` object by adding (subtracting for negative values) values to its members.
+The specified values must be numeric if specified.
+
+The algorithm is such that:
+  1. the individual values are added to the existing values.
+  2. the range of `days` is ensured to be between 1 and 29-31 depending on the month by adjusting `month`
+  3. the range of `months` is ensured to be between 1 and 12 by adjusting the `years`.
+
+#### CivilDate.prototype.with(values: DateLike) : CivilDate
```

Comment:
Are the timeline-anchored objects intentionally missing `with` methods?

Reply:
Nope. Oversight